django_app/report_a_suspected_breach/views.py

```diff
@@ -438,3 +432,25 @@ def done(self, form_list: list[str], **kwargs: object) -> HttpResponse:
 
 class CompleteView(TemplateView):
     template_name = "report_a_suspected_breach/complete.html"
+
+
+class RequestVerifyCodeView(FormView):
+    form_class = SummaryForm
+    template_name = "report_a_suspected_breach/form_steps/request_verify_code.html"
+    success_url = reverse_lazy("report_a_suspected_breach:email_verify")
+
+    def form_valid(self, form: SummaryForm) -> HttpResponse:
+        reporter_email_address = self.request.session.get("reporter_email_address")
```

Comment:
should be a hard dict lookup I think, we want it to error here if the `reporter_email_address` is not in the session:

```suggestion
        reporter_email_address = self.request.session["reporter_email_address"]
```